kiwix-desktop.pro

```diff
@@ -150,6 +150,9 @@ RESOURCES += \
     resources/contentmanager.qrc \
     resources/style.qrc
 
+QMAKE_LUPDATE = /usr/bin/lupdate
+QMAKE_LRELEASE = /usr/bin/lrelease
```

Comment:
I don't know why we use the variable QMAKE_LUPDATE and QMAKE_LRELEASE.
This come from this PR #25 and as @Ordissimo said itself, it is more accustomed to gettext than Qt. As I am.

If we don't need it (define it just to use it later), it may be better to directly use the binary name late.
ie : `system(lupdate -locations relative ..)`
And this way, we would be more tolerant to tools install elsewhere than in `/usr/bin`.

Reply:
OK. Had somehow the expectation to improve that line to apply the hardcoded path only if undefined... but not sure about which language is exactly used in the `.pro` file.